types/schemas/prototypeApplication/data/ApplicationData.ts

```diff
@@ -58,25 +58,36 @@ export type PPApplicationData = FeeCarryingApplicationData & {
 };
 
 /**
- * TypeMap of PrimaryApplicationTypes to their specific ApplicationData models
+ * TypeMap of granular application types to their specific ApplicationData models
  */
 export interface ApplicationDataVariants {
-  ldc: FeeCarryingApplicationData;
-  pa: FeeCarryingApplicationData;
-  pp: PPApplicationData;
-  listed: NonFeeCarryingApplicationData;
-  landDrainageConsent: FeeCarryingApplicationData;
-  wtt: NonFeeCarryingApplicationData;
-  hedgerowRemovalNotice: NonFeeCarryingApplicationData;
+  'ldc.breachOfCondition': FeeCarryingApplicationData;
+  'ldc.existing': FeeCarryingApplicationData;
+  'ldc.listedBuildingWorks': NonFeeCarryingApplicationData;
+  'ldc.proposed': FeeCarryingApplicationData;
```

Comment:
Per recent testing, we can now correctly encode which types of LDCs carry a fee & which don't :heavy_check_mark: